_tour/implicit-conversions.md

```diff
@@ -20,13 +31,33 @@ Implicit conversions are applied in two situations:
 In the first case, a conversion `c` is searched for which is applicable to `e` and whose result type conforms to `T`.
 In the second case, a conversion `c` is searched for which is applicable to `e` and whose result contains a member named `m`.
 
+{% tabs implicit-conversions_2 class=tabs-scala-version %}
+{% tab 'Scala 2' for=implicit-conversions_2 %}
+
 If an implicit method `List[A] => Ordered[List[A]]` is in scope, as well as an implicit method `Int => Ordered[Int]`, the following operation on the two lists of type `List[Int]` is legal:
 
+{% endtab %}
+{% tab 'Scala 3' for=implicit-conversions_2 %}
+
+If an implicit method `List[A] => Ordered[List[A]]` is in scope, as well as a `given`instance of `Conversion[Int, Ordered[Int]]`, the following operation on the two lists of type `List[Int]` is legal:
+
+{% endtab %}
+{% endtabs %}
+
+{% tabs implicit-conversions_3 %}
+{% tab 'Scala 2 and 3' for=implicit-conversions_3 %}
```

Comment:
I think it looks better if this code example is duplicated in both of the above tabs, rather than making a separate tab group